src/PowerShell/Utilities/Utils.ts

```diff
@@ -0,0 +1,55 @@
+import * as os from 'os';
+
+import Constants from '../Constants';
+import ScriptBuilder from './ScriptBuilder';
+import PowerShellToolRunner from './PowerShellToolRunner';
+
+export default class Utils {
+    static setPSModulePath(azPSVersion: string = "") {
+        let modulePath: string = "";
+        const runner: string = process.env.RUNNER_OS || os.type();
+        switch (runner.toLowerCase()) {
+            case "linux":
+                modulePath = `/usr/share/${azPSVersion}:`;
+                break;
+            case "windows":
+            case "windows_nt":
+                modulePath = `C:\\Modules\\${azPSVersion};`;
+                break;
+            case "macos":
+            case "darwin":
+                throw new Error(`OS not supported`);
+            default:
+                throw new Error(`Unknown os: ${runner.toLowerCase()}`);
+        }
+        process.env.PSModulePath = `${modulePath}${process.env.PSModulePath}`;
```

Comment:
Would be good to log the PSModulePath here.

Reply:
When this function is called the first time, the value of the azPSVersion param is empty. And so the value of the PSModulePath variable will be the same if the function isn't called. Why do it ?

Reply:
In case of linux, Az module is present inside folder "/usr/share/". When I run a "Get-Module" command later in the script to get latest version of Az module, Get-Module says "module doesnt exist" because "/usr/share" is not present in the path by default. Hence based on runner, am adding the path which contains the Az module.

Reply:
oh ok. hmm... ok so from reading the code it isn't obvious why we are doing it. can you add a comment explaining this. Add it where the function is called the first time